### PR TITLE
style(ui): add warning about keeping seed private

### PR DESCRIPTION
**Pull request by seanyesmunt**
## Description:
Added a warning label in the `NewWalletSeed` component. This uses a new ui component called `WarningLabel`

## Motivation and Context:

For #522 

## How Has This Been Tested?

Manually tested on light and dark mode

### Before
screen shot 2018-11-19 at 1 38 30 am

### After
screen shot 2018-11-19 at 1 21 56 am
screen shot 2018-11-19 at 1 23 20 am

## Types of changes:

Style change

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [x] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: app/components/Onboarding/NewWalletSeed/NewWalletSeed.scss [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
       &:nth-child(1) {
         width: 15%;
-        text-align: center;
```
Comment (seanyesmunt) [inline app/components/Onboarding/NewWalletSeed/NewWalletSeed.scss]: I removed this so the left row of seed words is aligned with the left edge of the warning label box. I see the reason for centering it though. I think either looks ok.

**Comment by seanyesmunt** [comment]
Will add a test for this component and run the messages script.

**Comment by seanyesmunt** [comment]
@mrfelton I ran `yarn extract-messages` but it seems to have removed some messages. Any ideas why?

**Comment by mrfelton** [comment]
@seanyesmunt in the `next` the onboarding process has been fully refactored - so this will need to be reworked against that branch.

Also, please could you add this new element into the storybook as part of this PR? IO notived that you added a new colour as part of this. I'm not sure wether or not we want to introduce a new colour to the colour palette, but if we do then that should also be added to the theme definitions and to colour palette section of the storybook (these are both newly added in the `next` branch)

I have asked Ole to take a look from a design perspective.

**Comment by mrfelton** [comment]
Regarding the message strings - it looks like some of these were updated recently in master but the extract message command had not been run so some of the translation files were a little out of sync.

I have run `yarn extract-messages` against master and created a PR to resolve this issue here https://github.com/LN-Zap/zap-desktop/pull/916

**Comment by mrfelton** [comment]
@seanyesmunt I rebased this branch and updated this PR slightly to reuse our existing `Message` component for the message. This was merged in #1029

**Pull request closed without merging**